collections-dictionary/CompareTwoDictionaries/CompareTwoDictionariesBenchmark/Program.cs

```diff
@@ -0,0 +1,4 @@
+﻿using BenchmarkDotNet.Running;
+using CompareTwoDictionariesBenchmark;
+
+BenchmarkRunner.Run<CompareDictionariesBenchmark>();
```

Comment:
I think this is dead code since we are running the benchmark in the main console application.